common/src/models/error.rs

```diff
@@ -30,114 +31,107 @@ impl Display for ApiError {
+            ErrorKind::ProjectCorrupted => StatusCode::BAD_REQUEST,
             ErrorKind::ProjectHasResources(resources) => {
                 let resources = resources.join(", ");
                 return Self {
                     message: format!("Could not automatically delete the following resources: {}. Please reach out to Shuttle support for help.", resources),
                     status_code: StatusCode::INTERNAL_SERVER_ERROR.as_u16(),
-                }
-            }
-            ErrorKind::InvalidProjectName(err) => {
-                return Self {
-                    message: err.to_string(),
-                    status_code: StatusCode::BAD_REQUEST.as_u16(),
-                }
-            }
-            ErrorKind::InvalidOperation => (StatusCode::BAD_REQUEST, "The requested operation is invalid"),
-            ErrorKind::ProjectAlreadyExists => (StatusCode::BAD_REQUEST, "A project with the same name already exists"),
-            ErrorKind::OwnProjectAlreadyExists(message) => {
-                return Self {
-                    message,
-                    status_code: StatusCode::BAD_REQUEST.as_u16(),
-                }
+                };
             }
-            ErrorKind::InvalidCustomDomain => (StatusCode::BAD_REQUEST, "Invalid custom domain"),
-            ErrorKind::CustomDomainNotFound => (StatusCode::NOT_FOUND, "Custom domain not found"),
-            ErrorKind::CustomDomainAlreadyExists => (StatusCode::BAD_REQUEST, "Custom domain already in use"),
-            ErrorKind::Unauthorized => (StatusCode::UNAUTHORIZED, "Unauthorized"),
-            ErrorKind::Forbidden => (StatusCode::FORBIDDEN, "Forbidden"),
-            ErrorKind::NotReady => (StatusCode::INTERNAL_SERVER_ERROR, "Service not ready"),
-            ErrorKind::DeleteProjectFailed => (StatusCode::INTERNAL_SERVER_ERROR, "Deleting project failed"),
-            ErrorKind::CapacityLimit => (StatusCode::SERVICE_UNAVAILABLE, "Our server is at capacity and cannot serve your request at this time. Please try again in a few minutes."),
+            ErrorKind::InvalidProjectName(_) => StatusCode::BAD_REQUEST,
+            ErrorKind::InvalidOperation => StatusCode::BAD_REQUEST,
+            ErrorKind::ProjectAlreadyExists => StatusCode::BAD_REQUEST,
+            ErrorKind::OwnProjectAlreadyExists(_) => StatusCode::BAD_REQUEST,
+            ErrorKind::InvalidCustomDomain => StatusCode::BAD_REQUEST,
+            ErrorKind::CustomDomainNotFound => StatusCode::NOT_FOUND,
+            ErrorKind::CustomDomainAlreadyExists => StatusCode::BAD_REQUEST,
+            ErrorKind::Unauthorized => StatusCode::UNAUTHORIZED,
+            ErrorKind::Forbidden => StatusCode::FORBIDDEN,
+            ErrorKind::NotReady => StatusCode::INTERNAL_SERVER_ERROR,
+            ErrorKind::DeleteProjectFailed => StatusCode::INTERNAL_SERVER_ERROR,
+            ErrorKind::CapacityLimit => StatusCode::SERVICE_UNAVAILABLE,
```

Comment:
It is possible to combine multiple arms with  `A | B | C ... => StatusCode:: ...`, but I'm not sure if that makes it more or less readable.